### PR TITLE
Issue #49: Enabling multiple disable

### PR DESCRIPTION
**Pull request by dan9186**
To address #49

### CODE REVIEW

File: init [hunk 1]
```diff
@@ -305,21 +305,28 @@ $k=\"$global\"
...
-			[ "$LSECONDARY[$f]" = "$mod" ] && LSECONDARY[$f]=()
-		for f in {1..${#RPRIMARY[@]}};
-			[ "$RPRIMARY[$f]" = "$mod" ] && RPRIMARY[$f]=()
-		for f in {1..${#RSECONDARY[@]}};
-			[ "$RSECONDARY[$f]" = "$mod" ] && RSECONDARY[$f]=()
-		for f in {1..${#TITLE[@]}};
-			[ "$TITLE[$f]" = "$mod" ] && TITLE[$f]=()
-		for f in {1..${#EXTRA[@]}};
-			[ "$EXTRA[$f]" = "$mod" ] && EXTRA[$f]=()
+		foreach mod in $@; do
```
Comment (brimstone) [inline init]: `$@` probably needs quotes, just in case we ever get multi word module names.

**Comment by brimstone** [comment]
Other than nit pick. LGTM. How about you @itsamenathan?

**Comment by itsamenathan** [comment]
Same, quotes are always better in shell.

**Comment by dan9186** [comment]
Sounds like a good idea.  Added, anything else?

**Comment by brimstone** [comment]
LGTM

**Comment by dan9186** [comment]
@itsamenathan what say you? wanna give it a merge?

**Pull request merged**